web/src/features/hfiCalculator/pages/HfiCalculatorPage.tsx

```diff
@@ -128,20 +147,21 @@ const HfiCalculatorPage: React.FunctionComponent = () => {
   const setNewFireStarts = (
     areaId: number,
     dayOffset: number,
-    newFireStarts: FireStarts
+    newFireStarts: FireStartRange
   ) => {
     if (!isUndefined(result)) {
-      const newPlanningAreaFireStarts = cloneDeep(result.planning_area_fire_starts)
-      newPlanningAreaFireStarts[areaId][dayOffset] = { ...newFireStarts }
+      // TODO: Move the smart date higher up
```

Comment:
did you plan to move smart date in a later PR?

Reply:
yes/no - depend on what feedback I get on the smartdate. Yes if we keep it, not if we don't.

Reply:
Please make smart date a separate PR unless it's not possible.

Reply:
Oh totally - this PR is too monstrous anyway!